Sources/PaymentButtons/PaymentButtonSize.swift

```diff
@@ -17,14 +17,8 @@ public enum PaymentButtonSize: Int, CustomStringConvertible {
 
     var font: UIFont {
         switch self {
-        case .mini, .collapsed:
-            return PaymentButtonFont.secondaryFont
-
-        case .expanded:
-            return PaymentButtonFont.primaryFont
-
-        case .full:
-            return PaymentButtonFont.systemFont18
+        case .mini, .collapsed, .expanded, .full:
+            return PaymentButtonFont.paypalPrimaryFont
```

Comment:
I don't think we need this switch at all, can just do:
```swift
var font: UIFont {
    PaymentButtonFont.paypalPrimaryFont
}
```

Reply:
Thanks, I'll get that fixed